src/jit/inline.cpp

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "jitpch.h"
+#ifdef _MSC_VER
+#pragma hdrstop
+#endif
+
+static const char* InlineExplanations[] = 
+{
+    "unused initial",
+#define INLINE_OBSERVATION(name, type, description, impact, target) description,
+#include "inline.def"
+#undef INLINE_OBSERVATION
```

Comment:
Having the `#define` and `#undef` in separate files strikes me as odd. There are only 3 uses.

I've put the unused cases into the master list. Hopefully all our target compilers like trailing commas in enumerations (I think they do because if you look carefully I left one in the original `Impact` enum).